### PR TITLE
Added APIs Guru to helpers

### PR DESCRIPTION
**Pull request by dletorey**
# Thank you for opening a pull request to tiny-helpers.dev ❤️

Before you submit your PR, please take a look at the [contributing guidelines](https://github.com/stefanjudis/tiny-helpers/blob/master/CONTRIBUTING.md). :)

To make it easier for us to review and merge your PR, please make sure …

- [ ] you only add one (!) new helper per pull request.
- [ ] you have checked if an open PR already exists.
- [ ] the submitted website is focused on a single, development related issue.
- [ ] the `desc` field includes an "actionable sentence" (e.g. "Create something great" or "Transform something into something else").

Thank you! 🙇‍♂️

### CODE REVIEW

**Comment by stefanjudis** [comment]
Thanks @dletorey. This URL doesn't look correct, or? :)

**Comment by dletorey** [comment]
I have updated the url, although the screenshot has not updated.

File: helpers/apis-guru.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "APIs Guru",
+  "desc": "API discovery platform, find and add APIs which then lists where they can be found, docs, swagger tools and OpenAPI specs.",
```
Comment (stefanjudis) [inline helpers/apis-guru.json:3]:
```suggestion
  "desc": "Find valuable APIs.",
```

**Comment by stefanjudis** [comment]
Thanks for the contribution. 🙇‍♂️ FYI I'll adjust it a little bit though. :)

**Pull request merged**